tests/PHPStan/Analyser/data/bug-9208.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * @param int|non-empty-list<int> $id_or_ids
+ * @return non-empty-list<int>
+ */
+function f(int|array $id_or_ids): array
+{
+	if (is_array($id_or_ids)) {
+		\PHPStan\Testing\assertType('non-empty-list<int>', (array)$id_or_ids);
```

Comment:
The file needs to have namespace `Bug9208`.

Reply:
The file isn't referenced in NodeScopeResolverTest

Reply:
fixed